Revert "Bump Spark Wallet"

Kukks opened the pull request:
Reverts btcpayserver/btcpayserver-docker#448

After the updates the spark url from services gives a 502 

```
docker logs -t -f generated_clightning_bitcoin_spark_1
2021-04-16T06:07:52.747873175Z nc: unix connect failed: Connection refused
```

Kukks commented:
cc @shesek  @NicolasDorier  anything comes to mind with this?

shesek commented:
Hi, sorry for taking long to get to this!

The issue appears to be caused by a mechanism I implemented to ease the migration to per-network subdirectories (introduced in c-lightning v0.8.0). If the directory mounted in `/etc/lightning` didn't have the `lightningd.sqlite3` file but a subdirectory matching the name of the network does have it, it will use the subdirectory instead.

BTCPay's (somewhat unusual) configuration for c-lightning creates the `lightning-rpc` file in the main directory while the database file is in the subdirectory, which caused this detection mechanism to fail. I implemented a change in https://github.com/shesek/spark-wallet/commit/6729f847b6467d87ca6abd1e403da423e060fcf7 that makes spark work properly when `/etc/lightning` is the directory with the `lightning-rpc` file (and updated the docs to advise users to mount it like that).

I tested with BTCPay and verified that this fixes the issue you were experiencing (using the development docker image `shesek/spark-wallet:0.2-next-standalone`).

I will soon make an official release with this (plus some other goodies) and send a PR to update to it.

Kukks commented:
> Hi, sorry for taking long to get to this!
> 
> The issue appears to be caused by a mechanism I implemented to ease the migration to per-network subdirectories (introduced in c-lightning v0.8.0). If the directory mounted in `/etc/lightning` didn't have the `lightningd.sqlite3` file but a subdirectory matching the name of the network does have it, it will use the subdirectory instead.
> 
> BTCPay's (somewhat unusual) configuration for c-lightning creates the `lightning-rpc` file in the main directory while the database file is in the subdirectory, which caused this detection mechanism to fail. I implemented a change in [shesek/spark-wallet@6729f84](https://github.com/shesek/spark-wallet/commit/6729f847b6467d87ca6abd1e403da423e060fcf7) that makes spark work properly when `/etc/lightning` is the directory with the `lightning-rpc` file (and updated the docs to advise users to mount that way).
> 
> I tested with BTCPay and verified that this fixes the issue you were experiencing (using the development docker image `shesek/spark-wallet:0.2-next-standalone`).
> 
> I will soon make an official release with this (plus some other goodies) and send a PR to update to it.

Thanks @shesek :)

ampon0001 commented:
$ gh issue list

Showing 4 of 4 issues in cli/cli

#16 Improving interactions with protected branches
#14 PR commands on a detached head
#13 Support for GitHub Enterprise (enhancement)
#8 Add an easier upgrade command (bug)

ampon0001 commented:
$ gh pr status

Relevant pull requests in cli/cli

Current branch
There is no pull request associated with [fix-homepage-bug]

Created by you
You have no open pull requests

Requesting a code review from you
#100 Fix footer on homepage [fix-homepage-footer]
✓ Checks passing - Review pending

ampon0001 commented:
docker logs -t -f generated_clightning_bitcoin_spark_1
2021-04-16T06:07:52.747873175Z nc: unix connect failed: Connection refused
